Tests/UnitTests/Android/app/src/main/cpp/JNI.cpp

```diff
@@ -21,6 +21,9 @@ Java_com_jsruntimehost_unittests_Native_javaScriptTests(JNIEnv* env, jclass claz
 
     android::global::Initialize(javaVM, context);
 
+    Babylon::DebugTrace::EnableDebugTrace(true);
+    Babylon::DebugTrace::SetTraceOutput([](const char* trace) { printf("%s\n", trace); });
```

Comment:
Ideally should flush after printf to avoid buffering. Maybe use std::cout instead which has a flush function.

Reply:
Fixed.